framework/behaviors/SluggableBehavior.php

```diff
@@ -81,13 +83,13 @@ class SluggableBehavior extends AttributeBehavior
      */
     public $value;
     /**
-     * @var bool whether to generate a new slug if it has already been generated before.
+     * @var boolean whether to generate a new slug if it has already been generated before.
      * If true, the behavior will not generate a new slug even if [[attribute]] is changed.
      * @since 2.0.2
      */
     public $immutable = false;
     /**
-     * @var bool whether to ensure generated slug value to be unique among owner class records.
+     * @var boolean whether to ensure generated slug value to be unique among owner class records.
```

Comment:
This is wrong change. Should be `bool` because `boolean` can't be used as return type in PHP 7+.